validation/src/test/scala/hmda/validation/rules/lar/validity/V525Spec.scala

```diff
@@ -0,0 +1,26 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ Loan, LoanApplicationRegister }
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V525Spec extends LarEditCheckSpec with BadValueUtils {
+  property("HOEPA status must be 1, or 2") {
+    forAll(larGen) { lar =>
+      V525(lar) mustBe Success()
```

Comment:
Now that #313 is merged in, there are convenience methods for this. You can replace `V525(lar) mustBe Success()` with `lar.mustPass`. Similarly, line 21 can be `invalidLar.mustFail`.